tasks/xpack/security/elasticsearch-security-file.yml

```
@@ -1,7 +1,10 @@
 ---
-- name: set fact manage_file_users
-  set_fact: manage_file_users=es_users is defined and es_users.file is defined and es_users.file.keys() | list | length > 0
+- set_fact: manage_file_users=false
 
+- set_fact: manage_file_users=true
+  when: es_users is defined and es_users.file is define and es_users.file.keys() | list | length > 0
```

```suggestion
  when: es_users is defined and es_users.file is defined and es_users.file.keys() | list | length > 0
```

This fixes the currently failing test:

```
14:34:57        TASK [elasticsearch : set_fact] ************************************************
14:34:57        fatal: [localhost]: FAILED! => {"msg": "The conditional check 'es_users is defined and es_users.file is define and es_users.file.keys() | list | length > 0' failed. The error was: template error while templating string: no test named 'define'. String: {% if es_users is defined and es_users.file is define and es_users.file.keys() | list | length > 0 %} True {% else %} False {% endif %}\n\nThe error appears to have been in '/tmp/kitchen/elasticsearch/tasks/xpack/security/elasticsearch-security-file.yml': line 4, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- set_fact: manage_file_users=true\n  ^ here\n"}
14:34:57        
```

---

Fixed in https://github.com/elastic/ansible-elasticsearch/pull/609/commits/08512fc17ea49568a58f788d1ed998133a68fd3e